src/error/JwtStructureError.ts

```diff
@@ -0,0 +1,7 @@
+export class JwtStructureError extends Error {
+  constructor(actual: string) {
+    super(
+      `The Authorization header token should be a JWT consisting of three parts: a JOSE header; a JWS payload; and a JWS signature.\nActual: ${actual}\nExpected: 3`
```

Comment:
How likely is it for that to happen though? Are there cases where code might accidentally generate 2 or 4 parts?

Reply:
A JWT can be either a JWS (always 3 parts) or a JWE (I think always 5 parts).

Clarified in: https://github.com/solid/access-token-verifier/pull/20/commits/111414a8690722c3c36ca672c2ca90b92c29541b